src/ecredis_server.erl

```diff
@@ -182,6 +181,8 @@ reload_slots_map(State) ->
     NewState = connect_all_slots(State, SlotsMaps),
     create_eredis_pids_cache(NewState, SlotsMaps),
     NodeList = [SlotsMap#slots_map.node || SlotsMap <- SlotsMaps],
+    OldNodeList = State#state.node_list,
+    disconnect_old_nodes(State#state.cluster_name, OldNodeList, NodeList),
```

Comment:
function name feels like we will disconnect from all of the old nodes - but we'll only disconnect from the ones that are missing in the new node list. so maybe rename the function here? or compute the difference here and send it across.. 

what do you think?

Reply:
call it check_and_disconnect maybe?